[2024-10-02T18:58:58.393+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T18:58:58.425+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-03T00:00:00+00:00 [queued]>
[2024-10-02T18:58:58.695+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-03T00:00:00+00:00 [queued]>
[2024-10-02T18:58:58.696+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T18:58:58.712+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carregar> on 2024-01-03 00:00:00+00:00
[2024-10-02T18:58:58.719+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=6102) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T18:58:58.720+0000] {standard_task_runner.py:63} INFO - Started process 6112 to run task
[2024-10-02T18:58:58.720+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'carregar', 'scheduled__2024-01-03T00:00:00+00:00', '--job-id', '1245', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpkl5l50ia']
[2024-10-02T18:58:58.722+0000] {standard_task_runner.py:91} INFO - Job 1245: Subtask carregar
[2024-10-02T18:58:58.784+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-03T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T18:58:58.882+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='carregar' AIRFLOW_CTX_EXECUTION_DATE='2024-01-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-03T00:00:00+00:00'
[2024-10-02T18:58:58.885+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T18:58:58.941+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T18:58:58.942+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T18:58:58.950+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=carregar, run_id=scheduled__2024-01-03T00:00:00+00:00, execution_date=20240103T000000, start_date=20241002T185858, end_date=20241002T185858
[2024-10-02T18:58:58.974+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T18:58:58.989+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T18:58:58.991+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T19:25:30.360+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:25:30.408+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-03T00:00:00+00:00 [queued]>
[2024-10-02T19:25:30.791+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-03T00:00:00+00:00 [queued]>
[2024-10-02T19:25:30.793+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:25:30.822+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carregar> on 2024-01-03 00:00:00+00:00
[2024-10-02T19:25:30.834+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=9534) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T19:25:30.839+0000] {standard_task_runner.py:63} INFO - Started process 9559 to run task
[2024-10-02T19:25:30.838+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'carregar', 'scheduled__2024-01-03T00:00:00+00:00', '--job-id', '2277', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp21cyqhvt']
[2024-10-02T19:25:30.840+0000] {standard_task_runner.py:91} INFO - Job 2277: Subtask carregar
[2024-10-02T19:25:30.931+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-03T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T19:25:31.076+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='carregar' AIRFLOW_CTX_EXECUTION_DATE='2024-01-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-03T00:00:00+00:00'
[2024-10-02T19:25:31.080+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:25:31.225+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T19:25:31.229+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:25:31.251+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=carregar, run_id=scheduled__2024-01-03T00:00:00+00:00, execution_date=20240103T000000, start_date=20241002T192530, end_date=20241002T192531
[2024-10-02T19:25:31.308+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T19:25:31.336+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:25:31.360+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
